BaseApiBundle/Tests/Transformer/TransformerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OpenOrchestra\BaseApiBundle\Tests\Transformer;
+
+use OpenOrchestra\BaseApi\Transformer\TransformerManager;
+use Phake;
+
+/**
+ * Test TransformerTestCase
+ */
+class TransformerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * test TransformerManager class
+     */
+    public function testTransformerManager()
+    {
+        $name = 'first_transfomer';
+        $transformer = Phake::mock('OpenOrchestra\BaseApi\Transformer\TransformerInterface');
+        Phake::when($transformer)->getName()
+            ->thenReturn($name);
+        Phake::when($transformer)->setContext(Phake::anyParameters());
```

Comment:
You take a point, I added this line only to respect all steps of the function addTransformer.